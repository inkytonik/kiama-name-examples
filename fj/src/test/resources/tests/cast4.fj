class A extends Object {
  A() {
    super();
  }
}
class B extends Object {
  B() {
    super();
  }
}
// Stupid cast
(B) new A()
