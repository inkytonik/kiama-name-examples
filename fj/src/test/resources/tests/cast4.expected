src/test/resources/tests/cast4.fj:12:1:info: (B)new A() ...: B
(B) new A()
^
src/test/resources/tests/cast4.fj:12:5:info: new A(): A
(B) new A()
    ^
B
